### PR TITLE
{lib}[GCCcore/7.3.0] OpenJPEG 2.3.0

### PR DESCRIPTION
**Pull request by fizwit**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/o/OpenJPEG/OpenJPEG-2.3.0-GCCcore-7.3.0.eb [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# easybuild easyconfig
+#
+# John Dey jfdey@fredhutch.org
+#
+# Fred Hutchenson Cancer Research Center Seattle WA
+name = 'OpenJPEG'
+version = '2.3.0'
+
+easyblock = 'CMakeMake'
```
Comment (boegel) [inline easybuild/easyconfigs/o/OpenJPEG/OpenJPEG-2.3.0-GCCcore-7.3.0.eb]: @fizwit For consistency with existing easyconfig files, this should go above the `name =` line (with an empty line in between).

File: easybuild/easyconfigs/o/OpenJPEG/OpenJPEG-2.3.0-GCCcore-7.3.0.eb [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+sources = ['v%(version)s.zip']
+checksums = ['e0bd4ee9ab6e4f3a581fca5e01fbfb45a8ab9062baeddd1211741531b53fe07d']
+
+separate_build_dir = True
+
+builddependencies = [
+    ('binutils', '2.30'),
+    ('CMake', '3.11.4')
+]
+
```
Comment (boegel) [inline easybuild/easyconfigs/o/OpenJPEG/OpenJPEG-2.3.0-GCCcore-7.3.0.eb]: @fizwit Please include a custom `sanity_check_paths`, to check for specific binaries in `bin/` and libraries in `lib*/` (and perhaps also header files in `include`, etc.).

Just checking for a couple key files is sufficient, the check is not supposed to be exhaustive.

Comment (boegel) [reply easybuild/easyconfigs/o/OpenJPEG/OpenJPEG-2.3.0-GCCcore-7.3.0.eb]: @fizwit You overlooked this comment w.r.t. a custom `sanity_check_paths`?

File: easybuild/easyconfigs/o/OpenJPEG/OpenJPEG-2.3.0-GCCcore-7.3.0.eb (hunk 1 above)
Comment (fizwit) [reply easybuild/easyconfigs/o/OpenJPEG/OpenJPEG-2.3.0-GCCcore-7.3.0.eb]: extreme badness - forgetting the sanity_check_paths

File: easybuild/easyconfigs/o/OpenJPEG/OpenJPEG-2.3.0-GCCcore-7.3.0.eb [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+sanity_check_paths = {
+    'files': ['bin/opj_compress',
+              'bin/opj_decompress',
+              'bin/opj_dump',
+              'include/openjpeg-2.3/openjpeg.h',
```
Comment (boegel) [inline easybuild/easyconfigs/o/OpenJPEG/OpenJPEG-2.3.0-GCCcore-7.3.0.eb]: @fizwit Please avoid hardcoding (partial) version, fixed in https://github.com/fizwit/easybuild-easyconfigs/pull/4

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node3100.skitty.os - Linux centos linux 7.5.1804, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 2.7.5
See https://gist.github.com/869f24244ce4d2ccdee2c1c6cabfce21 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2532.golett.os - Linux centos linux 7.5.1804, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/d774bec0819326896840423857e97b6b for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
gligar03.gligar.os - Linux centos linux 7.5.1804, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/a177e726272ae264da0cff506334b4d6 for a full test report.

**Comment by boegel** [comment]
Going in, thanks @fizwit!

**Pull request merged**